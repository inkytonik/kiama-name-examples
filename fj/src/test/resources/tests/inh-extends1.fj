class B extends A {
  B() {
    super();
  }
}
