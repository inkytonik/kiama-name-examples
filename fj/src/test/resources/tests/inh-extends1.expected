src/test/resources/tests/inh-extends1.fj:1:17:error: superclass A is not defined
class B extends A {
                ^
